packages/common/src/webrtc/Peer.ts

```diff
@@ -139,13 +140,17 @@ export default class Peer {
   }
 
   private _config(): RTCConfiguration {
-    const { iceServers = [], iceTransportPolicy = 'all' } = this.options
+
+
+    const { iceServers = [], iceTransportPolicy = 'all', disableIceUDP = false } = this.options
```

Comment:
I have a doubt about the term `disableIceUDP`, because this is not meant to disable any UDP ICE candidate, but to ignore `iceServers` URLs with UDP transport.
I know it's just a name, but I'd propose `disableUdpIceServers` instead (with `Udp` instead of `UDP`).